pkg/operator/observe_config.go

```diff
@@ -24,9 +24,11 @@ import (
 	"k8s.io/client-go/util/workqueue"
 
 	operatorconfigclientv1alpha1 "github.com/openshift/cluster-kube-apiserver-operator/pkg/generated/clientset/versioned/typed/kubeapiserver/v1alpha1"
-	operatorconfiginformerv1alpha1 "github.com/openshift/cluster-kube-apiserver-operator/pkg/generated/informers/externalversions/kubeapiserver/v1alpha1"
+	operatorclientinformers "github.com/openshift/cluster-kube-apiserver-operator/pkg/generated/informers/externalversions"
 )
 
+type observeConfigFunc func(kubernetes.Interface, map[string]interface{}) (map[string]interface{}, error)
```

Comment:
second arg should a `rest.Config` from `k8s.io/client-go/rest`.  You don't know what other kinds of clients will eventually be required.